build.sh

```diff
@@ -42,9 +42,9 @@ function docker_build_with_version {
 function squash {
   # FIXME: We have to use the exact versions here to avoid Docker client
   #        compatibility issues
-  easy_install -q --user docker_py==1.7.2 docker-squash==1.0.1
+  easy_install -q --user docker_py==1.10.6 docker-squash==1.0.5
```

Comment:
Hms, I missed this ugly heritage :) .. could we rather remove the `easy_install` entirely and depend on system package?